resources/skins/nudus/html_single_message.html

```diff
@@ -0,0 +1,14 @@
+<div dir="auto">
+    <div class="mwrapper">
+    <section class="mhead">
+        <div style="float: right; margin: 10px;"><!--where should I put it?????-->%7</div>

```

Comment:
"Where should I put it?"

What sort of embedded images are used and what are they for?

In GitHub feeds it is just an authors avatar, is it used for something else?

Reply:
%7 - as stated in "plain" skin's comments:

> %7 - Attached pictures (composed from html_enclosure_image.html).

For each image attached to article, the code from `html_enclosure_image.html` is taken and tokens in it are replaced with image URL, etc. SO you should in your `html_enclosure_image.html` just display the image via `img` tag.

https://github.com/martinrotter/rssguard/blob/master/resources/skins/plain/html_enclosure_image.html

Reply:
https://github.com/martinrotter/rssguard/pull/512#discussion_r719068469

@martinrotter Yes, but what kind of images are those?

Reply:
Images which were "attached" to article via feed/article creator. Those are not necessarily the same images as included directly in body of article.

https://en.wikipedia.org/wiki/RSS_enclosure

Reply:
Well, I guess when I see one I'll see if I need to move it elsewhere.

Reply:
http://feeds.feedburner.com/techcrunch

Reply:
> http://feeds.feedburner.com/techcrunch

Sorry, those few articles which do have them are quite broken for me.

Reply:
Could you perhaps take screenshot?

Reply:
![image](https://user-images.githubusercontent.com/1255302/135402882-e6eefefc-0ed6-4ef2-a0f6-f2c3366dfc94.png)

Reply:
Oh, so it's always some small image, correct?

Reply:
No, the source image can be of any size. Also, the actual size of the thumbnail shown in preview is controlled by "Height of image attachments" setting which should be honored by the skin:
`<a href="%1"><img src="%1" alt="%2" style="height: %3px;" /></a>`
%1 URL of image, %2 MIME text, %3 height/width of image thumbnail.

Reply:
Right, but where the value of %3 is coming from? 

(This should have been my second question.)
https://github.com/martinrotter/rssguard/pull/512:
>  I have a few questions.

Reply:
Oh! Never mind.

Reply:
"Height of images attachments", right?

Reply:
It certainly so, but I still not sure whether i should put it below or above the ruler.

I guess I leave it as is for now.

I have added a screenshot.

Reply:
so i can merge?

Reply:
Hmmm, there are still may be some issues which I overlooked.

I guess I rather test it a bit more.

Reply:
I will rebase it if needed, no need to hurry.

Reply:
> are quite broken for me.

This is what they looked like:

![image](https://user-images.githubusercontent.com/62730422/135531844-e1bb9979-6843-4396-bb92-389884f14bf5.png)